doc/github-actions: documented 'setup-atlas' action (#2063)
ariga/atlas: doc/md/integrations/github-actions.mdx

| [ariga/atlas-action](https://github.com/ariga/atlas-action)        | CI for schema changes                               |
| [ariga/atlas-sync-action](https://github.com/ariga/atlas-sync-action)   | Sync your migration directory to Atlas Cloud (atlasgo.cloud) |
| [ariga/atlas-deploy-action](https://github.com/ariga/atlas-deploy-action) | Deploy versioned migrations from GitHub Actions    |

> | [ariga/setup-atlas](https://github.com/ariga/set-uatlas) | Install Atlas from a Github Action    |

Reviewer: ```suggestion
| [ariga/setup-atlas](https://github.com/ariga/setup-atlas) | Install Atlas from a GitHub Actions workflow    |
```

#### `cloud-token`

**Required** The Atlas Cloud token to use for authentication.

## `ariga/setup-atlas`

> The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install Atlas from a github action.

Reviewer: ```suggestion
The [ariga/setup-atlas](https://github.com/ariga/setup-atlas) action can be used to install the Atlas CLI from a GitHub Actions workflow.
```

Author: In the previous PR I used `Atlas CLI` and Mashraki suggested to use just `Atlas`

Let's keep it consistent? which one should I use?

Reviewer: `Atlas` , sorry.

Author: 👍 
Done

This is useful if your team wants to use Atlas via scripts or when you want to use features that are not exposed by other actions.

### Example

```yaml
name: Deploy schema changes
run-name: I'm using Atlas 🚀
on: [push]
jobs:
  use-atlas:
    runs-on: ubuntu-latest
    steps:
      - uses: ariga/setup-atlas@master
      - run: atlas version
      # - run: atlas schema apply ... 
```

### Reference

> The action supports the `Version` input:

Reviewer: Action inputs are commonly `kebab-cased`
```suggestion
The action supports the `version` input:
```

> #### `Version`

Reviewer 2: maybe give an example of the semantic versioning, such as:
`
with:
    Version: v0.13.2
`

> #### `Version`

Reviewer: ```suggestion
#### `version`
```
